data/interfaces.txt

```diff
@@ -17,6 +17,17 @@ color "medium" .5 .5 .5 0.
 color "dim" .25 .25 .25 0.
 color "faint" .1 .1 .1 0.
 
+color "panel background" .1 .1 .1 1.
+color "dialog backdrop" 0. 0. 0. .7
+color "conversation background" .125 .125 .125 1.
+color "map sidepanel background" .125 .125 .125 1.
+color "shop sidepanel background" .2 .2 .2 1.
+color "shop sidepanel footer" .3 .3 .3 1.
```

Comment:
I think "side panel" should be two words. "Sidebar" is a well-established word, but "sidepanel" is not.

Reply:
Perhaps these should be sidebar then instead of sidepanel? Name suggestions are always welcomed